wellies/templates/suite.py_t

```diff
@@ -8,83 +10,111 @@ import wellies as wl
 ROOT_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)))
 
 
+@dataclass
```

Comment:
Hey @gareth-j , don't you think we can add these two classes to wellies/host.py? This would allow us to use the new api on existing suites just changing yml's and the Config implementation.